pages/search.tsx

```diff
@@ -402,7 +402,7 @@ const Header = () => {
   }
 };
 
-const getVerifiedHelpText = (verifiedType: VerifiedType) => {
+const getVerifiedHelpText = (verifiedType?: VerifiedType) => {
```

Comment:
hvorfor optional

Reply:
Skjønte det ikke sjæl, men intellisensen ble iaffal sur uten. Skal få rydda bedre ved ledig tid. Enum-parsinga kan være stødigere, så slipper vi sånne 0-er-falsy-greier.

Reply:
Jeg skrev en rant på slack 🤡 takk for meg